src/errors/PasswordInvalidError.ts

```diff
@@ -1,9 +1,17 @@
+<<<<<<< HEAD
 export class PasswordInvalidError extends Error {
   public details?: string;
   public readonly code: number;
 
   constructor(message: string) {
     super(message);
     this.code = 400;
+=======
+import { ApolloError } from 'apollo-server-errors';
+
+export class PasswordInvalidError extends ApolloError {
```

Comment:
mesmos comentários do arquivo anterior: conflito, nome do arquivo seguindo o padrão e poderia ser um BadRequestError mesmo, porque se for ter um arquivo/classe para cada erro 400, pode ficar com muitos arquivos desnecessários (imagina o trabalho que ia ser para refatorar, se necessário).
Faz sentido?

Reply:
Feito.